### PR TITLE
Translation nette-rules fix

### PR DESCRIPTION
**Pull request by merglv**
Trnaslation for data-nette-rules messages shuld be connected to control translator not to form translator.

- bug fix

### CODE REVIEW

**Comment by merglv** [comment]
So, how i can disable translation only for specific control?

If i set ->setTranslator(null) to control. Its working for labels, standart error messages and values. I think its logic to have affect to error messages in data-nette-rules too.

I dont belive is intention when setTranslator method have affect for generation error message for php validation but no generation message to HTML.

**Comment by dg** [comment]
If it affects PHP validation, it is bug.

File: tests/Forms/Forms.idMask.phpt [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
```
Comment (LauraBruno0609) [inline tests/Forms/Forms.idMask.phpt]: Unlocking

**Comment by LauraBruno0609** [comment]
Declare instruct typo

**Pull request closed without merging**